tests/v2/validation/standalone/corral_generic_test.go

```diff
@@ -0,0 +1,69 @@
+package standalone
+
+import (
+	"testing"
+
+	"github.com/rancher/rancher/tests/framework/clients/corral"
+
+	"github.com/rancher/rancher/tests/framework/pkg/namegenerator"
+	"github.com/rancher/rancher/tests/framework/pkg/session"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+const (
+	defaultRandStringLength = 5
+)
+
+type CorralStandaloneTestSuite struct {
+	suite.Suite
+	session *session.Session
+}
+
+func AppendRandomString(baseClusterName string) string {
```

Comment:
Don't we already have this function?

Reply:
I should've clarified, the function is already present in `/tests/v2/validation/provisioning/config.go`, we should move it to a util package instead since it's not strictly provisioning related.